### ISSUE TITLE
How we can change breakpoint for flexgrid?

### ISSUE THREAD

**Issue by paraselixir** [roylee0704/react-flexbox-grid#161]
Is there any way we  can customise the media points for this grid system.

**Comment by thomasbritton** [comment]
I don't think you can. As this seems to be is just a wrapper around another css grid system called [flexboxgrid2](https://github.com/evgenyrodionov/flexboxgrid2) and of which is a dependancy here and it looks like the props are you pass just maps to a class name generated by flexboxgrid2
